PHP basics task 1-6 /pdf.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+require('./fpdf186/fpdf.php');
+
+// Creating an object of fpdf.
+$pdf = new FPDF();
+// Adding a new page.
+$pdf->AddPage();
+$pdf->SetFont('Arial', 'B', 12);
+$first_name = $_POST['First_Name'];
+$last_name = $_POST['Last_Name'];
+$number = $_POST['phone_number'];
+$email = $_POST['email'];
+$subjects = $_POST['comment'];
```

Comment:
if we are not doing any extra processing on the values, no need to unnecessarily define new variables.